sdk/attestation/pom.xml

```diff
@@ -0,0 +1,58 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-security-attestation</artifactId>
```

Comment:
This should be named `azure-security-attestation-service`

Reply:
Why is the MAA service "azure-security-attestation-service", but KeyVault and other services simply "azure-security-keyvault" (or "azure-storage-blob"?

Reply:
Ok, figured it out - sdk/attestation/pom.xml, not sdk/attestation/azure-security-attestaion/pom.xml.